tutorials/general/t8_tutorial_build_cmesh.h

```diff
@@ -0,0 +1,39 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
+
+  t8code is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  t8code is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with t8code; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+/** file t8_step8.h
+ */
+
+#ifndef T8_STEP8_H
+#define T8_STEP8_H
+
+#include <t8.h>
+
+T8_EXTERN_C_BEGIN ();
+
+/** This is the main program of this example.
+ */
+int                 t8_step8_main (int argc, char **argv);

```

Comment:
```suggestion
int                 t8_tutorial_build_cmesh_main (int argc, char **argv);
```

Reply:
If we choose the rename the "main-function", we will have to do it in the header file as well.